stylix/palette.nix

```
@@ -61,71 +40,25 @@ in {
       '';
     };
 
-    image = mkOption {
-      type = types.coercedTo types.package toString types.path;
-      description = mdDoc ''
+    wallpaper = mkOption {
+        type =
+        let
+          customTypes = config.lib.stylix;
+        in with types; oneOf [customTypes.static customTypes.animation customTypes.video customTypes.slideshow];
```

Probably neater to use a `with` here

```suggestion
        with types; with config.lib.stylix; oneOf [static animation video slideshow];
```